app/models/mapit_postcode_response.rb

```diff
@@ -0,0 +1,35 @@
+class MapitPostcodeResponse
+  attr_reader :location
+
+  def initialize(location)
+    @location = location
+  end
+
+  def gss
+    location["codes"]["gss"]
+  end
+
+  def area_name
+    location["name"]
+  end
+
+  def country
+    location["country_name"]
+  end
+
+  def england?
+    country == "England"
+  end
+
+  def scotland?
```

Comment:
I wonder if we will be able to drop these methods if we don’t implement a kind of offering for nations other than england. I can see code along the lines of;
```
<% unless response.england? %>
   <p>It looks like you're in <%= response.country %></p>
   <p>Please see the covid advice for that nation</p>
   <p><%= t('local_restrictions.links.#{response.country}') %></p>
<% end %>
```

Reply:
Which would mean we can remove the methods for the other nations and the tests as well

Reply:
Potential yes. It depends on how the content ends up being structured and whether we use the same view for the devolved nations, or if each has its own partial...

Reply:
💯